### DISCUSSION TITLE
How can i use wandb log text values?

### DISCUSSION THREAD

**Discussion by morestart** [Lightning-AI/pytorch-lightning#9593] [General]
I see wandb api can log text values, can pl support this? i don't find any demo in pl doc

**Comment by tchaton** [comment]
There is a PR currently being worked on by the Wandb Team: https://github.com/PyTorchLightning/pytorch-lightning/pull/9545. You can track progress there and share your opinions on the design.

**Reply by morestart** [reply]
great work~ thanks for your reply!
